zcash_client_backend/src/data_api.rs

```diff
@@ -236,12 +246,13 @@ pub trait WalletRead {
 /// decrypted and extracted from a [`CompactBlock`].
 ///
 /// [`CompactBlock`]: crate::proto::compact_formats::CompactBlock
-pub struct PrunedBlock<'a> {
+pub struct PrunedBlock<Nf> {
```

Comment:
This is also going to need the Orchard nullifier type.

Reply:
We'll handle that when we introduce Orchard.